octavia_f5/common/config.py

```diff
@@ -97,6 +97,9 @@ def setup_logging(conf):
                help=_("Use Cipher String for ciphers used in server TLS profiles")),
     cfg.StrOpt('default_client_ciphers', default=None,
                help=_("Use Cipher String for ciphers used in client TLS profiles")),
+    cfg.IntOpt('availability_timeout', default=5.0,
+               help=_("Time in seconds before a device is marked as offline."
+                      "This is only used by status manager, NOT for AS3 requests.")),
```

Comment:
The default healthcheck intervall is 3 seconds, this would queue up the periodic sync if a device timeouts.

Reply:
I could set `availability_timeout` to a smaller value of course, but it still automatically retries several times when running into a timeout, which is not what we want in this case. I don't really know what would be a good solution for this...

Reply:
Turns out using `requests.get()` circumvents the retry mechanism.